### DISCUSSION TITLE
how to handle (no detections)?

### DISCUSSION THREAD

**Discussion by yangboz** [ultralytics/yolov5#8237] [Q&A]
some time still has no detections as follow, how to handle  (no detections) before  the next step? any API to indicate this information?
mage 1/1: 1920x905 (no detections)
Speed: 59.3ms pre-process, 346.9ms inference, 0.4ms NMS per image at shape (1, 3, 640, 320)
 image 1/1: 1920x905 (no detections)
Speed: 59.3ms pre-process, 346.9ms inference, 0.4ms NMS per image at shape (1, 3, 640, 320)

Thanks.

**Comment by yangboz** [comment]
inside of https://github.com/ultralytics/yolov5/blob/master/models/common.py , line of 663 , it happens, any clue?

**Comment by glenn-jocher** [comment]
@yangboz what's your question?

**Reply by yangboz** [reply]
let's explain , how to get know where did  the non detection happen  before   the "(no detection)" string  displayed?

**Reply by yangboz** [reply]
currently our workaround is len(detections.pandas().xyxy) >=1
